v22.2/migration-overview.md

```diff
@@ -53,27 +50,30 @@ Consider the following CockroachDB attributes and best practices:
 
 For additional considerations specific to other databases and data formats, see the corresponding documentation linked in [Step 2. Move your data to CockroachDB](#step-2-move-your-data-to-cockroachdb).
 
-## Step 2. Move your data to CockroachDB
+### Unimplemented features
 
-We recommend [using AWS Database Migration Service (DMS) to migrate data](aws-dms.html) from any database, such as PostgreSQL, MySQL, or Oracle, to CockroachDB.
+CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and is largely compatible with PostgreSQL syntax. However, the following PostgreSQL features do not yet exist in CockroachDB:
```

Comment:
do not yet exist in ---> do not exist in

Unless there is a necessity here to imply we will implement the features

Reply:
Yeah, I think we do want to leave that implication as various things on this list are on the roadmap.